разметил/ стилизую header

LNKcode opened the pull request.

LNKcode committed:
разметил/ стилизую header

LNKcode committed:
Добавил наставника

LNKcode committed:
Подключил шрифты

LNKcode committed:
делаю popover (test-1)

Kartaviy-arteom commented on index.html:
```diff
@@ -1,12 +1,145 @@
+                          width="39"
+                          height="43"
+                        />
+                        <h2 class="popover-cart__title">
```
Уровни заголовков

Kartaviy-arteom commented on index.html:
```diff
@@ -1,12 +1,145 @@
+          <a href="device-index.html" class="logo navigation__logo">
+            <img src="images/logo.svg" alt="Логотип" />
+          </a>
+          <div class="navigation__container">
```
Если в наименованиях классов используется бэм он должен быть без ошибок, https://yoksel.github.io/html-tree/

Kartaviy-arteom commented on index.html:
```diff
@@ -1,12 +1,145 @@
+              method="GET"
+              class="form"
+            >
+              <input
+                type="search"
+                name="search"
+                id="search"
+                placeholder="Поиск по сайту"
+                class="form__search"
+              />
```
нет большого смысла тут давать отправлять пустую форму

Kartaviy-arteom commented on css/style.css:
```diff
@@ -0,0 +1,224 @@
+  overflow: hidden;
+}
+
+body {
+  font-family: "Rubik", sans-serif;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 30px;
+  color: #444444;
+  margin: 0;
+}
+
+.container {
+  width: 1240px;
+  margin: 0 auto;
+  padding: 0 40px;
+}
```
скорее всего будет замечание к https://up.htmlacademy.ru/htmlcss/36/criteries#TEST-03 "У каждой страницы есть минимальная ширина по фрейму.", несмотря на то что остальное выполнится

LNKcode committed:
Слайдер (test-1)

LNKcode committed:
slider (test-2)

Сделал супер крутую анимацию на кнопку! и сделал фокус-визин на инпут с ужасным цветом с макета.

Kartaviy-arteom commented on css/style.css:
```diff
@@ -48,6 +53,7 @@ body {
   line-height: 30px;
   color: #444444;
   margin: 0;
+  min-width: 1240px;
```
в критерии указана ширина по фрейму в фигма, это 1440px
